gcloud-java-contrib/README.md

```diff
@@ -7,6 +7,12 @@ Contents
 --------
 
  * [gcloud-java-nio](./gcloud-java-nio/): NIO Filesystem Provider for Google Cloud Storage.
+ * [gcloud-java-nio-example](./gcloud-java-nio-example/): How to add GCS NIO after the fact.
```

Comment:
Because there's only one example? I don't think the "use an external jar" needs more than one example.